README.md

```diff
@@ -1,6 +1,73 @@
 # Northcoders News API
 
-## Cloning this repo
+## Overview
 
-Create `.env.test` and `.env.development` files. 
-Set environment variable `PGDATABASE=(name of your database)`
\ No newline at end of file
+This project is an API designed to provide programmatic access to application data, mimicking the backend service of a real-world application like Reddit. It uses PostgreSQL as the database, and interactions with the database are managed using node-postgres.
+
+## Viewing this repo
+Link to hosted version of repo: `https://northcoder-news.onrender.com/`
+
+
+## Installation and Setup
+
+1. Clone the repository to your local machine:
+   git clone `<repository_url>`
```

Comment:
I'd also add this as a code block to make it look consistent with the rest of the readme